src/App.jsx

```diff
@@ -3,26 +3,61 @@ import './App.scss';
 import './styles/general.scss';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
+import { getTodos } from './api';
 
 class App extends React.Component {
   state = {
     todos: [],
     selectedUserId: 0,
   };
 
+  componentDidMount() {
+    getTodos()
+      .then(todos => {
+        this.setState({
+          todos: todos.data,
+        })
+      })
+  };
+
+  setQuery = (str) => {
+    this.setState({
+      query: str,
+    });
+    this.setState(state => ({
+      todos: state.todos,
+    }));
+  }
```

Comment:
why do you need to keep query in this component? You can have it in the TodoList and not to pass it anywhere